tests/test_requests.py

```diff
@@ -1452,3 +1451,8 @@ def test_vendor_aliases():
 
     with pytest.raises(ImportError):
         from requests.packages import webbrowser
+
+
+
+if __name__ == '__main__':
+    unittest.main()
```

Comment:
Don't add this to the bottom of the test file: these tests don't run properly in unittest, they need to be executed in py.test.